sdk/tools/gen_baseaddress.py

```diff
@@ -206,10 +207,14 @@
     'dllexport_test_dll1.dll',
     'dllexport_test_dll2.dll',
     'dllimport_test.dll',
+    'localspl_apitest.dll',
     'MyEventProvider.dll',
-    'w32kdll_2k3sp2.dll',
-    'w32kdll_ros.dll',
-    'w32kdll_xpsp2.dll',
+    'redirtest1.dll',
+    'redirtest2.dll',
+    'win32u_2k3sp2.dll',
```

Comment:
```suggestion
    'win32u.dll',
    'win32u_2k3sp2.dll',
```
And update win32u_ros CMakeLists.txt too

Reply:
Ok. Depending on how soon that future Win7 change is expected to happen, I would suggest to add a comment to remember why this (test) dll is treated differently, or to simply sync current handling of this dll too until things do change.